test/validate_test.js

```diff
@@ -1,11 +1,19 @@
+import * as helper from  './helper';
+import subject from '../';
+import request from 'superagent-promise';
+import assert from 'assert';
+import Promise from 'promise';
+import testing from 'taskcluster-lib-testing';
+//import 'superagent-hawk';
+//import 'superagent';
 suite("api/validate", function() {
-  require('superagent-hawk')(require('superagent'));
-  var request         = require('superagent-promise');
-  var assert          = require('assert');
-  var Promise         = require('promise');
-  var subject         = require('../');
-  var helper          = require('./helper');
-  var testing         = require('taskcluster-lib-testing');
+   require('superagent-hawk')(require('superagent'));
+   //var request         = require('superagent-promise');
+   //var assert          = require('assert');
+   //var Promise         = require('promise');
+   //var subject         = require('../');
+   //var helper          = require('./helper');
+   //var testing         = require('taskcluster-lib-testing');
```

Comment:
did these get missed?

Reply:
I commented them instead of removing just to keep the original imports in sight. 
I will remove the comments once the error is resolved